mmdet/models/dense_heads/solov2_head.py

```diff
@@ -554,8 +554,7 @@ def loss_by_feat(self, mlvl_kernel_preds: List[Tensor],
         num_pos = 0
         for img_pos_masks in pos_masks:
             for lvl_img_pos_masks in img_pos_masks:
-                num_pos += lvl_img_pos_masks.count_nonzero()
-
+                num_pos += lvl_img_pos_masks.sum()
```

Comment:
Are these two operations equivalent and is it safe?

Reply:
I have checked. Because the type of `lvl_img_pos_masks` is torch.bool, count_nonzero is equal to sum.
Right now, I haven't have other idea to fix this line

Reply:
We can leave comments to indicate what this line of code does so that potential bug can be easily found if the type of lvl_img_pos_masks is changed.